### PR TITLE
Add docs for `pubsub.azure.servicebus.queues` component and update metadata for all other Service Bus components

### LINKED ISSUE #2912: [1.10] Add docs for `pubsub.azure.servicebus.queues` component and update metadata for all other Service Bus components
The pages for the Service Bus components need to be updated:

- dapr/components-contrib#2211 is adding a new component `pubsub.azure.servicebus.queues`, and renaming the existing `pubsub.azure.servicebus` to `pubsub.azure.servicebus.topics` (old name remains as alias)
- dapr/components-contrib#2203 changed the metadata options for the 3 Service Bus components (the two above and `bindings.azure.servicebusqueues`)

### PR DESCRIPTION
**Pull request by ItalyPaleAle**
Fixes #2912

### CODE REVIEW

File: daprdocs/content/en/reference/components-reference/supported-pubsub/setup-azure-servicebus-queues.md [hunk 1]
```diff
@@ -69,26 +68,25 @@ The above example uses secrets as plain strings. It is recommended to use a secr
 |--------------------|:--------:|---------|---------|
```
Comment (msfussell) [inline daprdocs/content/en/reference/components-reference/supported-pubsub/setup-azure-servicebus-queues.md:70]: Looking at the docs here https://docs.dapr.io/reference/components-reference/supported-pubsub/setup-azure-servicebus/ the third column "Details" is narrow in width and makes this page very long to scroll down. Could you make this column wider and "steal" from the other columns in width?

File: daprdocs/content/en/reference/components-reference/supported-pubsub/setup-azure-servicebus-topics.md [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+> __NOTE:__ The above settings are shared across all topics that use this component.
+
+{{% alert title="Warning" color="warning" %}}
+The above example uses secrets as plain strings. It is recommended to use a secret store for the secrets as described [here]({{< ref component-secrets.md >}}).
+{{% /alert %}}
+
+## Spec metadata fields
+
+| Field              | Required | Details | Example |
+|--------------------|:--------:|---------|---------|
```
Comment (msfussell) [inline daprdocs/content/en/reference/components-reference/supported-pubsub/setup-azure-servicebus-topics.md:75]: Looking at the docs here https://docs.dapr.io/reference/components-reference/supported-pubsub/setup-azure-servicebus/ the third column "Details" is narrow in width and makes this page very long to scroll down. Could you make this column wider and "steal" from the other columns in width?

File: daprdocs/content/en/reference/components-reference/supported-pubsub/setup-azure-servicebus-queues.md [hunk 3]
```diff
@@ -1,14 +1,17 @@
...
+description: "Detailed documentation on the Azure Service Bus Queues pubsub component"
 aliases:
-  - "/operations/components/setup-pubsub/supported-pubsub/setup-azure-servicebus/"
+  - "/operations/components/setup-pubsub/supported-pubsub/setup-azure-servicebus-queues/"
 ---
 
 ## Component format
-To setup Azure Service Bus pubsub create a component of type `pubsub.azure.servicebus`. See [this guide]({{< ref "howto-publish-subscribe.md#step-1-setup-the-pubsub-component" >}}) on how to create and apply a pubsub configuration.
+
+To setup Azure Service Bus Queues pubsub create a component of type `pubsub.azure.servicebus.queues`. See [this guide]({{< ref "howto-publish-subscribe.md#step-1-setup-the-pubsub-component" >}}) on how to create and apply a pubsub configuration.
```
Comment (msfussell) [inline daprdocs/content/en/reference/components-reference/supported-pubsub/setup-azure-servicebus-queues.md:12]: I would be useful to indicate any benefits/differences between using ASB Queues over ASB Topics. Why would you choose one over the other? Maybe it is just including this link https://learn.microsoft.com/en-us/azure/service-bus-messaging/service-bus-queues-topics-subscriptions as a reference. Given Queues in new in this release, some overview here would be valuable IMO. WDYT?

File: daprdocs/content/en/reference/components-reference/supported-pubsub/setup-azure-servicebus-queues.md (hunk 1 above)
Comment (ItalyPaleAle) [reply daprdocs/content/en/reference/components-reference/supported-pubsub/setup-azure-servicebus-queues.md:70]: I can't control the width of the columns with Markdown. But I changed the example to "see above"

File: daprdocs/content/en/reference/components-reference/supported-pubsub/setup-azure-servicebus-topics.md (hunk 2 above)
Comment (ItalyPaleAle) [reply daprdocs/content/en/reference/components-reference/supported-pubsub/setup-azure-servicebus-topics.md:75]: Same as above

File: daprdocs/content/en/reference/components-reference/supported-pubsub/setup-azure-servicebus-queues.md (hunk 3 above)
Comment (ItalyPaleAle) [reply daprdocs/content/en/reference/components-reference/supported-pubsub/setup-azure-servicebus-queues.md:12]: The link was actually already there, on the next line! But I made it more clear that it explains the difference

**Comment by hhunter-ms** [comment]
@ItalyPaleAle could you resolve merge conflicts?

**Comment by ItalyPaleAle** [comment]
Done @hhunter-ms

**Review by hhunter-ms — APPROVED** [review]

**Pull request merged**